### PR TITLE
Added php 8.2 edge

### PR DESCRIPTION
**Pull request by GeNyaa**
Since alpine doesn't have a stable version of PHP 8.2 yet I marked it as edge as has been done before.

### CODE REVIEW

File: files/php82-edge/etc/php82/php-fpm.conf [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+pm = ondemand
+pm.max_children = 5
+pm.process_idle_timeout = 10s
+pm.max_requests = 500
+
+; Health check
+ping.path = /php-fpm-ping
+
+; Include other configuration files
+include=/etc/php81/php-fpm.d/*.conf
```
Comment (eXistenZNL) [inline files/php82-edge/etc/php82/php-fpm.conf]: This needs to be php82 too, would you mind updating?

**Comment by eXistenZNL** [comment]
Oh man what a coincidence, I was JUST talking with a colleague about creating a PHP 8.2 existenz/builder container and now you open this PR :D Thanks for your contribution man!

**Comment by GeNyaa** [comment]
@eXistenZNL Updated the file, should be all set now.
Love the container, simple and does the job. 👍

**Comment by GeNyaa** [comment]
@eXistenZNL any update on merging this?

**Comment by eXistenZNL** [comment]
Oh wow, i was on a MASSIVE holiday spree far away from anything digital. Let's see here!

**Comment by eXistenZNL** [comment]
And merged, thanks a ton!

**Pull request merged**